mod/research/set/self/research_page/params.rb

```diff
@@ -162,7 +162,7 @@ def researchable_answer?
 
   def existing_answer_with_source?
     existing_answer? && researchable_answer? &&
-      (!answer_card.hybrid? || answer_card.calculation_overridden?)
+      (!answer_card.hybrid? || answer_card.researched_value?)
```

Comment:
I find this a little tricky to read, but I spent a few minutes trying to make it easy and basically failed.

Reply:
maybe if the second line were:
`(answer_card.researched? || answer_card.researched_value?)`